src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        setColor(color);
+        this.radius = radius;
+        setArea(Math.PI * radius * radius);
```

Comment:
```suggestion
        super(color);
        this.radius = radius;
```
- better call the constructor to initialize color in the parent class
- we shouldn't save the area to the field because after each modification of radius value we should re-calculate the area.